src/queue.ts

```diff
@@ -31,7 +32,7 @@ export class Queue<TItem> extends EventEmitter {
     }).promise()
   }
 
-  startProcessing (handler : (item : TItem) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
+  startProcessing (handler : (item : TItem, message : any) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
```

Comment:
@grvcoelho I can help you set the typings in the project. Can we do that monday, in the office?